scholarx/past-programs/2020/index.html

```diff
@@ -446,12 +450,57 @@ <h5>Projects :</h5>
             url: 'https://script.google.com/macros/s/AKfycbx_JpPBVggdoBq2wO6NDaP0uvTb9-VZiJF77GbnfUSSQFNdBZ8a7KZKuXEvJXdB2m-FXQ/exec',
             dataType: 'json',
             success: function (data) {
-                let templateMentees = Mustache.render($("#templateMentees").html(), {"data": data});
-                //display mentees
-                $("#teamMentees").html(templateMentees);
+                document.getElementById('btnMentees').style.visibility = "visible";
+                //slice array to two parts
+                if (data.length >= 6) {
+                    let tempProfileCount =6;
+                    let profileCount =data.length;
+                    let initialPart = data.slice(0, 6);
+
+                    //mustache render - initial part
+                    let initialProfileSet = Mustache.render(
+                        $('#templateMentees').html(), { 'data': initialPart }
+                    );
+
+                    //display first 6 mentees
+                    $('#teamMentees').html(initialProfileSet);
+                    $('#mentee-show-less').hide();
+
+                    //display 6 more mentees with a click
+                    $('#mentee-show-more').click(function () {
+                        tempProfileCount+=6;
+                        let nextPart =data.slice(tempProfileCount-6,tempProfileCount);
+
+                        //mustache render - next part
+                        let nextProfileSet = Mustache.render(
+                            $('#templateMentees').html(), { 'data': nextPart });
+                        $(nextProfileSet).appendTo('#teamMentees').hide().fadeIn(1000);
+                        $("#mentee-show-less").show();
+
+                        if(tempProfileCount>=profileCount){
+                            $('#mentee-show-more').hide();
+                        }
+                    });
+
+                    $("#mentee-show-less").click(function () {
```

Comment:
Do we have a similar code snippet somewhere else? @gajindubandara

Reply:
A similar code is used in the `scholarx/past-programs/2019/index.html`. Shall I Make a generalized function to load mentees on the `assets/js/functions.js `  ? @anjula-sack